src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -78,6 +78,9 @@
     <PackageReference Include="Microsoft.Azure.WebJobs.Host.Storage" Version="5.0.0-beta.1" />
     <PackageReference Include="Microsoft.Azure.WebSites.DataProtection" Version="2.1.91-alpha" />
     <PackageReference Include="Microsoft.IdentityModel.Tokens" Version="$(IdentityDependencyVersion)" />
+    <PackageReference Include="ServerlessSecurity" Version="1.0.0" />
```

Comment:
Shouldn't this include be "ServerlessSecurity$(RuntimeIdentifier)" as it is here: https://github.com/Azure/azure-functions-host/compare/dev...veerashayyagari:azure-functions-host:veerash/privateruntimehost-linux

Reply:
Yeah was testing only Windows with Generic one from local. Update it based on RuntimeIdentifier